streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -683,7 +700,7 @@ public void shouldCheckpointOffsetsOnCommit() throws IOException {
         task.initializeTopology();
         task.commit();
         final OffsetCheckpoint checkpoint = new OffsetCheckpoint(new File(stateDirectory.directoryForTask(taskId00),
-                                                                          ProcessorStateManager.CHECKPOINT_FILE_NAME));
```

Comment:
nit: move `new File` to new line for consistent formatting (note, that `ProcessorStateManager.CHECKPOINT_FILE_NAME)` is second parameter of `File` constructor.